### ISSUE TITLE
Run custom uninstall if uninstaller is not present on Mac

### ISSUE THREAD

**Issue by MartinIngesen** [botherder/flexikiller#1]
I'm not sure if this is a possible scenario, but if the current Mac script (`kill.sh`) doesn't find the `Uninstall.sh` under `/usr/libexec/.blblu/blblu/Contents/Resources/`, it will simply fail with a

> I can't find the uninstall script!

Would it be an idea to reuse the Uninstall.sh that's in this repo already, and run that if the "`if [ -f $UNINSTALL ]; then`" fails?

**Comment by botherder** [comment]
Yes possibly, and I thought about it, but I wouldn't know why the uninstall utility wouldn't be there.
